web/libs/editor/src/components/HtxTextBox/HtxTextBox.jsx

```diff
@@ -178,12 +182,32 @@ export class HtxTextBox extends React.Component {
       <>
         <Paragraph {...props}>
           <span ref={this.textRef}>{text}</span>
-          {isEditable && onChange && (
-            <EditOutlined onClick={this.startEditing} aria-label="Edit Region" className="ant-typography-edit" />
-          )}
         </Paragraph>
+        {isEditable && onChange && (
+        <Tooltip title="Edit">
+          <Elem
+            tag={Button}
+            name="edit"
+            type="text"
+            className="button__icon edit"
+            aria-label="Edit Region"
+            onClick={this.startEditing}
+            icon={<IconEdit />}
+          />
+        </Tooltip>
```

Comment:
I'm surprised linting didn't pick this up but i expected an extra tab in this block

Reply:
ahh i see linter check is failing - you can get those fixed automatically by running `make fmt-all` in application root directory